.travis.yml

```diff
@@ -15,7 +15,7 @@ install:
   - chmod +x /home/travis/.local/bin/rpmdev-packager
 script:
   - tox
-  - docker build --build-arg VERSION=master -t quay.io/foreman/obal:latest .
+  - docker build --build-arg VERSION=$TRAVIS_COMMIT -t quay.io/foreman/obal:latest .
```

Comment:
That doesn't seem to work. Maybe we try HEAD ?

Reply:
The problem is that it's a PR reference which isn't fetched by default. That's why it can't check it out.